### PR TITLE
Removing the Github token from KMM's job.

### PR DESCRIPTION
**Pull request by ybettan**
This token isn't working for what we need.

Signed-off-by: Yoni Bettan <yonibettan@gmail.com>

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kernel-module-management/kernel-module-management-presubmits.yaml [hunk 1]
```diff
@@ -10,22 +10,9 @@ presubmits:
       description: "make sure each PR to kernel-module-management has a single commit."
     spec:
       containers:
-      - image: ghcr.io/supportpal/github-gh-cli
+      - image: bitnami/git:latest
```
Comment (qbarrand) [inline config/jobs/kubernetes-sigs/kernel-module-management/kernel-module-management-presubmits.yaml:13]: How about using the [Kubernetes image](https://github.com/kubernetes/test-infra/tree/master/images/git)?

Comment (ybettan) [reply config/jobs/kubernetes-sigs/kernel-module-management/kernel-module-management-presubmits.yaml]: Fixed.

**Comment by ybettan** [comment]
@mrunalp Do you know if we have any action item here?

Regarding the Warnings:
```
{
  "component": "checkconfig",
  "file": "k8s.io/test-infra/prow/logrusutil/logrusutil.go:128",
  "func": "k8s.io/test-infra/prow/logrusutil.ThrottledWarnf",
  "level": "warning",
  "msg": "deprecated field: maintainers_id is configured for repo_milestone, maintainers_team should be used instead",
  "severity": "warning",
  "time": "2022-09-08T09:14:55Z"
}
```

Looks like we are using the [default-maintainer](https://github.com/kubernetes/test-infra/blob/f1fe3ba01f21be7ddbaf74341c50c7b48e906068/config/prow/plugins.yaml#L483-L489) so I don't think we have any action here.

```
{
  "component": "checkconfig",
  "file": "k8s.io/test-infra/prow/cmd/checkconfig/main.go:91",
  "func": "main.reportWarning",
  "level": "warning",
  "msg": "[error unmarshaling config/jobs/kubernetes/generated/generated.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes/sig-cloud-provider/gcp/registry/default.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"buildImageRegistry\", error unmarshaling config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes/sig-testing/conformance-e2e.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"skip_branches\", error unmarshaling config/jobs/kubernetes/sig-testing/make-test.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"allowPrivilegeEscalation\", error unmarshaling config/jobs/kubernetes/test-infra/test-infra-trusted.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-csi/external-attacher/external-attacher-config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-csi/external-provisioner/external-provisioner-config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-csi/external-resizer/external-resizer-config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-csi/external-snapshotter/external-snapshotter-config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-csi/livenessprobe/livenessprobe-config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-csi/node-driver-registrar/node-driver-registrar-config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-0-3.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"volumes\", error unmarshaling config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-0-4.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"volumes\", error unmarshaling config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-0.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"volumes\", error unmarshaling config/jobs/kubernetes-sigs/cluster-api-operator/cluster-api-operator-presubmits-main.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"env\", error unmarshaling config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-postsubmits.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"requests\", error unmarshaling config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-sigs/gcp-filestore-csi-driver/gcp-filestore-csi-driver-config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\", error unmarshaling config/jobs/kubernetes-sigs/kind/kind.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"always_run\", error unmarshaling config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"optional\", error unmarshaling config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"always_run\", error unmarshaling config/jobs/kubernetes-sigs/wg-multi-tenancy/hnc-e2e.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"always_run\", error unmarshaling config/jobs/kubernetes-sigs/wg-multi-tenancy/mtb-presubmit.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"privileged\"]",
  "severity": "warning",
  "time": "2022-09-08T09:14:55Z"
}
```

I don't see anything specific for the `kernel-module-management` repo either here.

```
{
  "component": "checkconfig",
  "file": "k8s.io/test-infra/prow/cmd/checkconfig/main.go:94",
  "func": "main.reportWarning",
  "level": "fatal",
  "msg": "Strict is set and there were warnings",
  "severity": "fatal",
  "time": "2022-09-08T09:14:55Z"
}
```

This one most probably comes from the 2 first warnings.

**Pull request merged**